oceania/README.OCEANIA

```diff
@@ -86,3 +86,134 @@ of the grid file will give results agreeing to within 1cm 99.97% of the time.
 Uses AusGeoid2020 model.
 
 * AUSGeoid2020_20180201.gtx
+
+[Creative Commons Attribution 4.0]: https://creativecommons.org/licenses/by/4.0/
+[GDA2020 Technical Manual]: http://www.icsm.gov.au/datum/gda2020-and-gda94-technical-manuals
+[ICSM]: https://github.com/icsm-au/transformation_grids
+
+## New Zealand Grids
+
+### NZGD1949 to NZGD2000 transformation 
```

Comment:
It is a bit confusing to have that one here, as it belongs to the main proj-datumgrid package. If you want to keep it there, then that should be mentionned

Reply:
Given that NZGD49 has been deprecated for nearly 20 years I don't think this needs to be in the base proj-datumgrid.  It is one of the less useful NZ grids.  I thought you had reservations about removing it (possibly as it is referenced in some of the documentation).  So for now maybe I should just add that as a note to the README as you suggest.

Reply:
>  I thought you had reservations about removing it

yes, because people who would use it have been expecting it to be in main and only one package for the last 20 years or so... So moving it could break their workflows, which we try to avoid in the PROJ 6.x series. PROJ 6.3 will likely be the last version to use that structure of proj-datumgrid + regional packages. Next generation grid repository candidate is in https://github.com/osgeo/proj-datumgrid-geotiff